shell/platform/android/io/flutter/view/TextureRegistry.java

```diff
@@ -45,6 +52,9 @@ interface SurfaceTextureEntry {
 
     /** Set a listener that will be notified when the most recent image has been consumed. */
     default void setOnFrameConsumedListener(@Nullable OnFrameConsumedListener listener) {}
+
+    /** Set a listener that will be notified when a memory pressure warning was forward. */
+    default void setOnLowMemoryListener(@Nullable OnLowMemoryListener listener) {}
```

Comment:
can this be called `setOnTrimMemoryListener`?

Reply:
done.